apps/api-extractor-model/src/items/ApiItem.ts

```diff
@@ -215,6 +234,27 @@ export class ApiItem {
   public getSortKey(): string {
     return this.containerKey;
   }
+
+  /**
+   * PRIVATE
+   *
+   * @privateRemarks
+   * Allows ApiItemContainerMixin to assign the parent.
+   *
+   * @internal
+   */
+  public [ApiItem_setParent](parent: ApiItem | undefined): void {
```

Comment:
We have a lint rule that prevents `apiItem["_parent"]` access because if the member gets renamed and we forget to rename the string literal, the compiler won't catch that.

I really should rename this to something like "onAddToContainer()` since in the current design, there is supposed to be only one specific way in which it can get set.

Reply:
I renamed it to `ApiItem_onParentChanged`